spring-integration-core/src/test/java/org/springframework/integration/config/xml/HeaderFilterParserTests-context.xml

```diff
@@ -2,23 +2,30 @@
 <beans:beans xmlns="http://www.springframework.org/schema/integration"
 	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
 	xmlns:beans="http://www.springframework.org/schema/beans"
-	xsi:schemaLocation="http://www.springframework.org/schema/beans
-			http://www.springframework.org/schema/beans/spring-beans.xsd
-			http://www.springframework.org/schema/integration
-			http://www.springframework.org/schema/integration/spring-integration.xsd">
+	xmlns:context="http://www.springframework.org/schema/context"
+	xmlns:util="http://www.springframework.org/schema/util"
+	xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+		http://www.springframework.org/schema/context http://www.springframework.org/schema/context/spring-context-4.2.xsd
+		http://www.springframework.org/schema/integration http://www.springframework.org/schema/integration/spring-integration.xsd
+		http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util-4.2.xsd">
```

Comment:
```
Execution failed for task ':spring-integration-core:checkTestConfigs'.

> Hardcoded XSD version in the config files:

  spring-integration-core/src/test/java/org/springframework/integration/config/xml/HeaderFilterParserTests-context.xml

  Please, use versionless schemaLocations for Spring XSDs to avoid issues with builds on different versions of dependencies.
```
Travis sees everything... :smile: